docker/tomcat/entrypoint.sh

```diff
@@ -18,6 +18,8 @@ password@javax.sql.BaseDataSource=${DB_ENV_MYSQL_PASSWORD}
 
 search.solr.server=http://solr:8983/solr/search
 ses.solr.server=http://solr:8983/solr/ses
+shoal.solr.url=http://solr:8983/solr/shoal
+oxam.solr.url=http://solr:8983/solr/oxam
```

Comment:
No, but it's a bug in the current setup, never going back so not worth another jira.